src/main/resources/assets/treechopper/lang/zh_CN.lang

```diff
@@ -1,21 +0,0 @@
-proxy.printBlock=����:
```

Comment:
lets keep chinese language :)

Reply:
I wanted to ask you about that, my IDE won't open the chinese lang files, I think it's because it doesn't understand the encodings.  I'm not sure if it's corrupted. I tried telling it UTF-8, UTF-16, windows-1252. Do you know how to edit the file? I can change the structure to json, but none of my editors will open the file.

Reply:
No file is fine. You need Simplified Chinese (GBK) encoding. Check here: https://github.com/ondrejduchon/TreeChopper/blob/master/src/main/resources/assets/treechopper/lang/zh_CN.lang

Reply:
I tried that, but it still wouldn't work.  I edited it manually in github's web UI and it appears to work.  I also added one additional translation string, "command.choice" which will need translations in other languages.  It is a feedback message to the player who changed the config that gives a success message.

Reply:
@hybcloud Hello, could you please revise that everything is correct?